python/sagemaker-multumodel-endpoint/requirements-dev.txt

```diff
@@ -0,0 +1 @@
+pytest==6.2.5
```

Comment:
I don't think this is used anywhere. If so, would you mind getting rid of this file?

Reply:
Removed requireements-dev.txt